android-sdk/src/main/java/com/checkout/android_sdk/Input/YearInput.kt

```diff
@@ -64,15 +64,15 @@ class YearInput(internal var mContext: Context, attrs: AttributeSet? = null) :
 
         onItemSelectedListener = object : OnItemSelectedListener {
```

Comment:
Please check the Country input as well it may have similar problem.

Reply:
Verified. Applied null check at the usability of this component.